tests/dir_helpers.py

```diff
@@ -178,32 +188,36 @@ class PosixTmpDir(TmpDir, pathlib.PurePosixPath):
     pass
 
 
+@pytest.fixture(scope="session")
+def make_tmp_dir(tmp_path_factory, request):
```

Comment:
How about having a `raw_dir` that does no magic at all (with above suggestions https://github.com/iterative/dvc/pull/3197/files#r368376497), but only gives you an empty directory?

Reply:
Sorry, looks like this was the intention of the PR. We can have a separate `raw_dir` as well then. :slightly_smiling_face:

Reply:
There is a `tmp_path` pytest standard fixture.

Reply:
This won't work when we need several of those. This is like `tmp_path` and `tmp_path_factory` from pytest.